projects/kit/components/input-slider/test/input-slider-new.component.spec.ts

```diff
@@ -1,548 +0,0 @@
-import {Component, TemplateRef, ViewChild} from '@angular/core';
-import {ComponentFixture, TestBed} from '@angular/core/testing';
-import {FormControl, ReactiveFormsModule} from '@angular/forms';
-import {CHAR_NO_BREAK_SPACE, TuiContextWithImplicit} from '@taiga-ui/cdk';
-import {TuiRootModule, TuiTextfieldControllerModule} from '@taiga-ui/core';
-import {TuiInputSliderComponent, TuiInputSliderModule} from '@taiga-ui/kit';
-import {configureTestSuite, TuiNativeInputPO, TuiPageObject} from '@taiga-ui/testing';
-
-@Component({
-    template: `
-        <tui-root>
-            <tui-input-slider
-                new
-                [formControl]="control"
```

Comment:
Github shows that this file is deleted. It not true.

I've deleted the file with all legacy props tests. And rename 
```
projects/kit/components/input-slider/test/input-slider-new.component.spec.ts
```
⬇️ 
```
projects/kit/components/input-slider/test/input-slider.component.spec.ts
```